src/app/app.component.ts

```diff
@@ -1,8 +1,16 @@
-import { Component } from '@angular/core';
+import { AuthUserService } from './shared/services/auth-user.service';
+import { Component, OnDestroy } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {}
+export class AppComponent implements OnDestroy {
+
+  constructor(private authUserService: AuthUserService) {}
+
+  ngOnDestroy(): void {
+    this.authUserService.clearTokens();
+  }
```

Comment:
As I understand, you want to clear tokens on application destroy. That means if you accidentally close your page - all tokens will be removed and you will need to relogin again. I believe you do not need this logic here

Reply:
Deleted it, i will try to do log out soon